client/src/templates/Challenges/classic/classic.css

```diff
@@ -11,9 +11,9 @@
   top: -999em;
 }
 
-.monaco-editor .editor-widget {
+/* .monaco-editor .editor-widget {
   display: none !important;
-}
+} */
```

Comment:
A couple of thoughts.

1. If we are going to remove this CSS, we should actually remove it - not just comment it out. Commenting out code is how code bases get unnecessarily bloated.

2. Before we can remove this, we need to decide what to do about the `Ctrl+H` shortcut - that's the find+replace and opens the same widget.

Reply:
1. Makes sense, will change as per decision on 2.
2. Check out my comment below!